fizz-buzz.test.js

```diff
@@ -4,3 +4,32 @@ const { fizzBuzz } = require('./fizz-buzz');
 // 2. add a first test. You could use one of the console.log tests in the other file as a starting point
 // 3. run the test with `npm test` to see if it passes
 // 4. add another test, and another, and another, until you've tested all the functionality of the fizzBuzz function
+
+describe('fizz', () => {
+
+      test('should if the number is divisible by 3, return Fizz', () => {
+            const result = fizzBuzz(3);
+            expect(result).toEqual('Fizz')
+      })
+})
+describe('Buzz', () => {
```

Comment:
Usually, you'd group several tests together in a single `describe()`. The label for the describe could be the name of the function eg `fizzBuzz`.